### DISCUSSION TITLE
Option to deactivate unused integrations

### DISCUSSION THREAD

**Discussion by JoghurtMitDerEcke** [outline/outline#6367] [Self Hosting] [answered]
Hi,

is there an option to deactivate unused integrations? We use very few integrations and the list would be clearer if many of the integrations were deactivated.

Thank you very much,
Benjamin

**Comment by tommoor** [comment, marked as answer]
Hey, there isn't such option at the moment. In what sense are the unused integrations getting in the way?

**Comment by JoghurtMitDerEcke** [comment]
Thanks for your message @tommoor 

It's not a big problem for us. I thought I would just ask if there is such an option.

But to explain: It would simply be helpful if the list did not consist of so many entries (integrations). It would then be clearer and we could simply show our employees only the integrations we need.

Does that make sense to you?

**Reply by tommoor** [reply]
Yes, it makes sense. Will keep in mind for a future setting – if someone sees this and wants to work on it I'm happy to supervise an approach

**Reply by JoghurtMitDerEcke** [reply]
Thank you!

**Comment by vijfhoek** [comment]
I would also like to see this happen, as we for example have no interest in using Otter.ai for privacy reasons, and don't want the option to be there for our users for this reason.

**Comment by entenhausen1** [comment]
+1

Hi @tommoor ,

My team and I are testing Outline as a possible Confluence DC alternative for our company, and one piece of feedback was, "Can we deactivate all the macros we don't need?"

`..and what about the air-gapped-version - it's impossible to use the external macros/integrations - are they still showing up?
